Microsoft.Azure.Cosmos/src/FeedResponseBinder.cs

```
@@ -47,28 +47,20 @@ public static DocumentFeedResponse<T> Convert<T>(DocumentFeedResponse<dynamic> d
         /// DEVNOTE: Need to refactor to use CosmosJsonSerializer
         /// </summary>
         public static DocumentFeedResponse<T> ConvertCosmosElementFeed<T>(
-            DocumentFeedResponse<CosmosElement> dynamicFeed,
+            QueryResponse dynamicFeed,
             ResourceType resourceType,
             JsonSerializerSettings settings)
         {
             if (dynamicFeed.Count == 0)
             {
-                return new DocumentFeedResponse<T>(
-                new List<T>(),
-                dynamicFeed.Count,
-                dynamicFeed.Headers,
-                dynamicFeed.UseETagAsContinuation,
-                dynamicFeed.QueryMetrics,
-                dynamicFeed.RequestStatistics,
-                dynamicFeed.DisallowContinuationTokenMessage,
-                dynamicFeed.ResponseLengthBytes);
+                throw new NotImplementedException();
```

Shouldn't we be returning an empty list in this case?

---

The v2 pipeline doesn't have the conversion logic from v3 to v2. v3 types are missing several required properties.  I would rather it fail fast to ensure none of the tests are dependent on this old code path.